### PR TITLE
TCCP: Select multiple situations, placeholder situation content

### PR DESCRIPTION
**Pull request by chosak**
This PR modifies the TCCP "Explore credit cards for your situation" landing page by allowing users to select more than one situation at a time. The situation content on the landing page has also been refreshed.

The results view now also shows situation content relevant to the selected situations. This content is placeholder and will be updated.

## How to test this PR

Run a local server and visit http://localhost:8000/consumer-tools/credit-cards/explore-cards/. Experiment with selecting one or more situations to see how the results view appears.

This branch is now also live on (internal) DEV4.

## Notes and todos

There are a few minor frontend tweaks that I could use help from @contolini. First, the icons on the select view don't exactly line up with the text as they should:

image

I tried experimenting with a few CSS tweaks but couldn't figure this out. Additionally, I'm not super-confident in how I implemented the CSS classes so any feedback or improvements would be appreciated!

The content here also needs to be updated, although that can happen in a future PR.

## Checklist

- [x] PR has an informative and human-readable title
- [x] Changes are limited to a single goal (no scope creep)
- [x] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)

### CODE REVIEW

**Comment by caheberer** [comment]
I noticed on the list view that the icons for "pay less" and "pay more" are black. Average is gold, though. 🤔

**Comment by chosak** [comment]
@caheberer thanks! Just fixed.

**Comment by contolini** [comment]
Nice work! I pushed up a commit for the icon issue. Icon alignment in list items is an issue throughout the site, e.g. in the mega menu:

image

File: cfgov/unprocessed/apps/tccp/css/main.less [hunk 1]
```diff
@@ -38,6 +38,37 @@
...
+.a-checkbox:checked + .a-label > .a-label_helper {
+  display: grid;
+  grid-column-start: 1;
+  grid-column-end: 3;
+  margin-top: 1em;
+  margin-bottom: 1em;
+}
+
+.a-label_helper .m-list {
+  list-style: square;
```
Comment (contolini) [inline cfgov/unprocessed/apps/tccp/css/main.less:54]: @caheberer the designs show circle bullet points but I made them squares to match the [DS guidance](https://cfpb.github.io/design-system/components/lists#bulleted-list).

**Review by contolini — APPROVED** [review]

**Comment by chosak** [comment]
Thank you @contolini!

> let's shorten the rating labels to remove "than average"

@caheberer I've made this change to shorten the labels only in the table.

**Pull request merged**